lib/Workflow/Exception.pm

```diff
@@ -105,6 +105,7 @@ sub _massage {
 
     my %params = ( ref $items[-1] eq 'HASH' ) ? %{ pop @items } : ();
     my $msg = join '', @items;
+    $msg =~ s/\n/\\n/g; # don't log newlines as per Log4perl recommendations
```

Comment:
@ehuelsmann is it better to remove the newlines characters all together?

```suggestion
    $msg =~ s/\n//g; # don't log newlines as per Log4perl recommendations
```

Reply:
I've contemplated that. I thought it was impractical to do in case the logs contain a stack dump.

I'm definitely not against, but I propose to use `s/\n/ /g` instead, because the newline removal may cause line ending and start to be concatenated.